core/src/epicli/data/common/ansible/playbooks/roles/image_registry/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Check if image is already loaded
+  shell: "docker images {{ specification.repository_image.name }} --format {{ '{{' }}.ID{{ '}}' }}"
```

Comment:
FYI: there is also docker Ansible module

Reply:
Yes there is, but require to install another python dependency on machine that it will be used. Think it's not worth it.